packages/dx-react-scheduler-material-ui/src/templates/date-navigator/root.test.jsx

```diff
@@ -1,5 +1,6 @@
 import * as React from 'react';
 import { createShallow } from '@material-ui/core/test-utils';
+import { mount } from 'enzyme';
```

Comment:
We should use only [material-ui mout](https://material-ui.com/guides/testing/#createmount-options-mount) here.

Reply:
OK

Reply:
I've replaced enzyme mount with material-ui mount